src/test/java/org/gridsuite/modification/server/modifications/TwoWindingsTransformerCreationTest.java

```diff
@@ -0,0 +1,265 @@
+                                        .x(0)
+                                        .g(0)
+                                        .b(0)
+                                        .build(),
+                                TapChangerStepCreationInfos.builder()
+                                        .index(6)
+                                        .rho(1)
+                                        .r(0)
+                                        .x(0)
+                                        .g(0)
+                                        .b(0)
+                                        .build(),
+                                TapChangerStepCreationInfos.builder()
+                                        .index(7)
+                                        .rho(1)
+                                        .r(0)
+                                        .x(0)
+                                        .g(0)
+                                        .b(0)
+                                        .build(),
+                                TapChangerStepCreationInfos.builder()
+                                        .index(8)
+                                        .rho(1)
+                                        .r(0)
+                                        .x(0)
+                                        .g(0)
+                                        .b(0)
+                                        .build()
+                        ))
+                        .build())
+                .build();
+    }
+
+    @Override
+    protected MatcherTwoWindingsTransformerCreationInfos createMatcher(ModificationInfos modificationInfos) {
+        return MatcherTwoWindingsTransformerCreationInfos.createMatcherTwoWindingsTransformerCreationInfos((TwoWindingsTransformerCreationInfos) modificationInfos);
+    }
+
+    @Override
+    protected void assertNetworkAfterCreation() {
+        assertNotNull(getNetwork().getTwoWindingsTransformer("new2wt"));
+    }
+
+    @Override
+    protected void assertNetworkAfterDeletion() {
+        assertNull(getNetwork().getTwoWindingsTransformer("new2wt"));
+    }
+
+    @SneakyThrows
+    @Test
+    public void testCreateWithErrors() {
+        TwoWindingsTransformerCreationInfos twoWindingsTransformerCreationInfos = (TwoWindingsTransformerCreationInfos) buildModification();
+        twoWindingsTransformerCreationInfos.setEquipmentId("");
+        String substationCreationInfosJson = mapper.writeValueAsString(twoWindingsTransformerCreationInfos);
+        MvcResult mvcResult = mockMvc.perform(post(getNetworkModificationUri()).content(substationCreationInfosJson).contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().is5xxServerError()).andReturn();
+        String resultAsString = mvcResult.getResponse().getContentAsString();
+        assertEquals(resultAsString, new NetworkModificationException(NetworkModificationException.Type.CREATE_TWO_WINDINGS_TRANSFORMER_ERROR, "Invalid id ''").getMessage());
+    }
```

Comment:
in previous unit test, there is an error test case with BUS_NOT_FOUND.
can we have it here too ?